File: ClientApp/SuspendedStorefront/src/app/api/models/charity-product.ts
/* tslint:disable */
/* eslint-disable */
import { Charity } from './charity';
import { Product } from './product';
export interface CharityProduct {
  charity?: Charity;
  charityID?: string;
  id?: string;
  isActive?: boolean;
  lastDelivery?: string;
  product?: Product;
  productID?: string;
  weeklyQuantity?: number;
}

File: api/Models/CharityProduct.cs
namespace SuspendedStorefront.Models
{
    public class CharityProduct
    {
        public Guid ID { get; set; }
        public Guid CharityID { get; set; }

        public Guid ProductID { get; set; }

        public int WeeklyQuantity { get; set; }

        public bool IsActive { get; set; }

        public DateTime LastDelivery { get; set; }

        public Charity Charity { get; set; }
        public Product Product { get; set; }
    }
}